index.ftd

```diff
@@ -27,7 +28,6 @@ string title:
 string company-name:
 ftd.image-src logo:
 optional string contact-1:
-optional string contact-2:
```

Comment:
Please don't remove this field `contact-2`. And also use this field in your component definition